lib/pure/strmisc.nim

```diff
@@ -85,6 +86,78 @@ proc rpartition*(s: string, sep: string): (string, string, string)
 
   return partition(s, sep, right = true)
 
+
+func parseFloatThousandSep*(str: string; sep = ','; decimalDot = '.'): float {.since: (1, 5).} =
```

Comment:
can this take `openArray[char]` ?

Reply:
But why?, `parseFloat` takes `string`. :thinking:

Reply:
it avoids un-necessary copies in some cases

Reply:
You mean 2 overloads?.